nc/source/ui/editor/EditorControls.h

```diff
@@ -0,0 +1,212 @@
+#pragma once
+#ifdef NC_EDITOR_ENABLED
+#include "Profile.h"
+#include "Ecs.h"
+#include "graphics/d3dresource/GraphicsResourceManager.h"
+#include "imgui/imgui.h"
+
+
+namespace nc::ui::editor::controls
+{
+    const auto TitleBarHeight = 40.0f;
```

Comment:
Not sure if worth it for IMGUI, but we should eventually not hardcode these sizes.

Reply:
Agreed. Once we've settled on a permanent solution, we'll figure out a proper way to do layout and scaling.